tutorials/introductory/usage.py

```diff
@@ -175,77 +170,373 @@
+l, = ax.plot(x, np.cumsum(data2), color='orange', linewidth=2)
+l.set_linestyle(':')
 
-    param_dict : dict
-       Dictionary of keyword arguments to pass to ax.plot
+###############################################################################
+# Colors
+# ------
+#
+# Matplotlib has a very flexible array of colors that are accepted for most
+# artists; see the :doc:`colors tutorial </tutorials/colors/colors>` for a
+# list of specifications. Some Artists will take multiple colors.  i.e. for
+# a `~.Axes.scatter` plot, the edge of the markers can be different colors
+# from the interior:
 
-    Returns
-    -------
-    out : list
-        list of artists added
-    """
-    out = ax.plot(data1, data2, **param_dict)
-    return out
+fig, ax = plt.subplots(figsize=(5, 2.7))
+x = np.arange(len(data1))
+ax.scatter(data1, data2, s=50, facecolor='C0', edgecolor='k')
 
 ###############################################################################
-# which you would then use as:
+# Linewidths, linestyles, and markersizes
+# ---------------------------------------
+#
+# Line widths are typically in typographic points (1 pt = 1/72 inch) and
+# available for artists that have stroked lines.  Similarly, stroked lines
+# can have a linestyle.  See the :doc:`linestyles example
+# </gallery/lines_bars_and_markers/linestyles>`.
+#
+# Marker size depends on the method being used.  `~.Axes.plot` specifies
+# markersize in points, and is generally the "diameter" or width of the
+# marker.  `~.Axes.scatter` specifies markersize as approximately
+# proportional to the visual area of the marker.  There are also an array of
+# markerstyles available as string codes (see :mod:`~.matplotlib.markers`) or
+# users can define their own `~.MarkerStyle` (see
+# :doc:`/gallery/lines_bars_and_markers/marker_reference`):
 
-data1, data2, data3, data4 = np.random.randn(4, 100)
-fig, ax = plt.subplots(1, 1)
-my_plotter(ax, data1, data2, {'marker': 'x'})
+fig, ax = plt.subplots(figsize=(5, 2.7))
+ax.plot(data1, 'o', label='data1')
+ax.plot(data2, 'd', label='data2')
+ax.plot(data3, 'v', label='data3')
+ax.plot(data4, 's', label='data4')
+ax.legend()
+
+################################################################################
+#
+# Labelling plots
+# ===============
+#
+# Axes labels and text
+# --------------------
+#
+# `~.Axes.set_xlabel`, `~.Axes.set_ylabel` and `~.Axes.set_title` are used to
+# add text in the indicated locations (see :doc:`/tutorials/text/text_intro` for
+# more discussion).  Text can also be directly added to plots using
+# `~.Axes.text`:
+
+mu, sigma = 100, 15
+x = mu + sigma * np.random.randn(10000)
+fig, ax = plt.subplots(figsize=(5, 2.7), constrained_layout=True)
+# the histogram of the data
+n, bins, patches = ax.hist(x, 50, density=1, facecolor='C0', alpha=0.75)
+
+ax.set_xlabel('Smarts')
+ax.set_ylabel('Probability')
+ax.set_title('Histogram of IQ')
+ax.text(60, .025, r'$\mu=100,\ \sigma=15$')
```

Comment:
```suggestion
ax.set_xlabel('Value')
ax.set_ylabel('Probability')
ax.set_title(r'Histogram of normal distribution')
ax.text(60, .025, r'$\mu=100,\ \sigma=15$')
```

Reply:
Changed to aardvark length (with a caveat that the distribution was not real, though I changed the mean to be correct for aardvarks).